### ISSUE TITLE
Excel Reports - Unclear dependency message

### ISSUE THREAD

**Issue by Puckfist** [infiniteautomation/ma-core-public#485]
Without mango's API module, navigating to the Excel Reports page yields large black letters proclaiming:

Required module '{0}' does not exist

**Comment by terrypacker** [comment]
This is a 2 part fix that we will do after the 2.7.0 release.

Part 1 in core, change
``` common.moduleDoesNotExist=Required module '{0}' does not exist ```
to
``` common.moduleDoesNotExist=Required module {0} does not exist ```

Part 2 in module, change line 295 in excelReports.jsp from
``` <fmt:message key="common.moduleDoesNotExist"> ```
to
```
 <h1 style="color:red">
    <fmt:message key="common.moduleDoesNotExist">
      <fmt:param value="mangoApi"/>
    </fmt:message>
  </h1>

```

**Comment by Puckfist** [comment]
resolved
